compose.yml

```diff
@@ -1,5 +1,25 @@
 name: captable
 services:
+  app:
+    image: node:${NODE_VERSION}-alpine
+    container_name: captable-app
+    depends_on:
+      - pg
+      - smtp
+      - minio
+    restart: always
+    ports:
+      - 3000:3000
+    env_file:
+      - ".env"
+    working_dir: /app
+    command: [/bin/sh, -c, "npm i -g pnpm@9.1.2 && pnpm install && pnpm dev"]
+    volumes:
+      - .:/app
+      - /app/node_modules
+      - /app/.next
+      - /app/.pnpm-store
```

Comment:
I dont see reference to `Dockerfile.dev`, does this mean we dont need it anymore? Let's delete the file

Reply:
Yeah, `Dockerfile.dev` is not needed anymore, and the reason for it is to simplify things and not care about the `Dockerfile.dev` altogether.